src/WebJobs.Extensions.NotificationHub/Bindings/NotificationHubAsyncCollector.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.NotificationHubs;
+
+namespace Microsoft.Azure.WebJobs.Extensions.NotificationHubs
+{
+    internal class NotificationHubAsyncCollector : IAsyncCollector<Notification>
+    {
+        INotificationHubClientService _clientService;
+        string _tagExpression;
+
+        public NotificationHubAsyncCollector(INotificationHubClientService clientService, string tagExpression)
```

Comment:
I'd remove 'Client' from this interface name. 'Service' is a bit of an overloaded term here but to me it means 'this thing interacts with NotificationHubs'. It'll just shorten this up a little.